### PR TITLE
Fixed poster list layout

### PR DESCRIPTION
**Pull request by mic4ael**

### CODE REVIEW

File: indico/MaKaC/webinterface/tpls/PosterTemplates.tpl [hunk 1]
```diff
@@ -14,7 +14,7 @@
     % for template in templateList:
     <li>
         <span class="name">${template['name']}</span>
-        <div class="toolbar right thin inline">
+        <div class="right thin inline">
```
Comment (ThiefMaster) [inline indico/MaKaC/webinterface/tpls/PosterTemplates.tpl]: This change results in the buttons showing up separately instead of tied together...

Comment (mic4ael) [inline indico/MaKaC/webinterface/tpls/PosterTemplates.tpl]: Indeed, however is it necessary to put them so close to each other?

Comment (ThiefMaster) [inline indico/MaKaC/webinterface/tpls/PosterTemplates.tpl]: On this page? Not necessarily. But it does look better in toolbar style.

To me the original problem looks like there's some clearing missing or some margins are too low.

Comment (ThiefMaster) [inline indico/MaKaC/webinterface/tpls/PosterTemplates.tpl]: `clear:both` on the `<li>` seems to avoid the problem without removing the toolbar style

Comment (mic4ael) [inline indico/MaKaC/webinterface/tpls/PosterTemplates.tpl]: Ok, thanks for the suggestion.

File: indico/MaKaC/webinterface/tpls/PosterTemplates.tpl [hunk 2]
```diff
@@ -12,7 +12,7 @@
 <div class="template_list">
   <ul>
     % for template in templateList:
-    <li>
+    <li style="clear: both;">
```
Comment (ThiefMaster) [inline indico/MaKaC/webinterface/tpls/PosterTemplates.tpl]: `class="clear"` would work, too (defined in _core.scss:58)

**Comment by ThiefMaster** [comment]
s/Fixed/Fix/ in the commit message please

**Comment by ThiefMaster** [comment]
Merged 37f0238

**Pull request closed without merging**